CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## vN.N.N
 * issue #1237: CMS - Gallery Launch Text not reflecting with red background.
+* issue #1233: CMS - Instagram embed preview not rendering.
```

Comment:
nit: make changelog entries in numerical order

Reply:
@cohencodes fixed